Irina/src/main/java/FirstTasks/DisplayMonth.java

```diff
@@ -0,0 +1,49 @@
+package FirstTasks;
+/*
+Display the month by it number
+ */
+
+public class DisplayMonth {
+    public static void main(String[] args) {
+        int month = 13;
```

Comment:
what is the output of switch in this case?

Reply:
Done